pkg/machines/components/libvirtSlate.jsx

```diff
@@ -107,7 +85,7 @@ class LibvirtSlate extends React.Component {
                         <input type="checkbox"
                                id="enable-libvirt"
                                disabled={!name}
-                               checked={this.state.libvirtEnabled}
+                               checked
```

Comment:
This looks a bit curious. As long as long as there still actually is a checkbox, the component should have state for it and set it here. Just the default state should change.

Reply:
Ups, good catch. Removed that by mistake.

Reply:
Ups, indeed. I removed it by mistake.